.circleci/config.yml

```diff
@@ -8,18 +8,19 @@ version: 2.1
 jobs:
   build:
     docker:
-      - image: openmcworkshop/paramak:dependencies
-
-    working_directory: ~/repo
-
-    steps:
-      - checkout
-      - run:
-          name: install
-          command:
-            pip install .[neutronics]
-
-      # run tests!
+      - image: ukaea/paramak:dev
```

Comment:
shouldn't be the ubuntu image here ?

Reply:
ah yes sorry I wasn't sure if this is the  tag you want to apply to the image or the name of the image to use